edge_api/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM amazoncorretto:18-alpine3.13-jdk
+EXPOSE 8050
+ARG JAR_FILE=target/*.jar
+COPY ${JAR_FILE} app.jar
+ENTRYPOINT ["java","-jar","/app.jar"]
```

Comment:
You need to add the environment variables (from the application properties file) here. This is needed so the container can find the other containers in the kubernetes cluster